templates/default/opensearch_dashboards.yml.erb

```diff
@@ -29,18 +29,18 @@ server.rewriteBasePath: false
 
 # The URLs of the Elasticsearch instances to use for all your queries.
```

Comment:
you should base on this file 
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/1.1/config/opensearch_dashboards.yml

Reply:
Done.